tests/test_emulator/test_emulator.py

```diff
@@ -59,6 +59,27 @@ def test_load_many_fluxes(self, mock_emulator):
         assert len(flux) == len(params)
         assert np.all(np.isfinite(flux))
 
+    def test_load_flux_norm(self, mock_hdf5_interface, mock_emulator):
+        params = mock_hdf5_interface.grid_points[0]
+        grid_flux = mock_hdf5_interface.load_flux(params)
+        np.random.seed(123)
+        emu_flux = mock_emulator.load_flux(params, norm=True)
+        assert np.allclose(emu_flux, grid_flux, rtol=1e-2)
```

Comment:
the relative tolerance in this test feels pretty high, although `load_flux` includes random number generation so perhaps there is a way to test this using some kind of ensemble statistic for the GP instead of a single draw. Thoughts @iancze ?

Reply:
I guess it depends on how many terms are involved in the PCA decomposition and what sort of stellar parameter range is being decomposed. Getting a pixel flux to < 1% could be pretty good if the number of PCA terms is relatively small and the stellar parameter range is substantial.

Reply:
yes, the number of components is pretty low here, less than 5, so this is probably good enough for a test